src/bls/bls.h

```diff
@@ -197,7 +202,7 @@ class CBLSWrapper
 
     inline bool CheckMalleable(Span<uint8_t> vecBytes, const bool specificLegacyScheme) const
     {
-        if (memcmp(vecBytes.data(), ToByteVector(specificLegacyScheme).data(), SerSize)) {
+        if (memcmp(vecBytes.data(), ToBytes(specificLegacyScheme).data(), SerSize)) {
```

Comment:
here's UB. after call data() temporary object is erased.

```suggestion
        const auto bytes{ToBytes(specificLegacyScheme)};
        if (memcmp(vecBytes.data(), bytes.data(), SerSize)) {
```